packages/cli/src/util/colorizer.ts

```diff
@@ -0,0 +1,22 @@
+import { mapValues, isArray } from 'lodash';
+import { Dictionary } from '@stoplight/types';
+
+type ParamValue = number | string | string[] | unknown;
+export type ValuesTransformer = (values: Dictionary<ParamValue>) => Dictionary<string | string[]>;
+
+export const PRE_PARAM_VALUE_TAG = 'PRE_PARAM_VALUE_TAG';
+export const POST_PARAM_VALUE_TAG = 'POST_PARAM_VALUE_TAG';
```

Comment:
@XVincentX, it's now `~pre~` and `~post~` since special chars were escaped by `URL.expand` - https://github.com/stoplightio/prism/blob/ba472b9288a287603fed252a482d08ee12f09dc5/packages/cli/src/util/paths.ts#L36